core/sds-aspect-model-document-generators/README.md

```diff
@@ -31,23 +31,40 @@ For more information on SPARQL construct queries, Graphviz and Mustache see:
 ## Document Generator
 
 The document generator creates an HTML page for an Aspect Model. The page includes the diagram for the Aspect Model.
-
-Embedding font and images for making the html page to run standalone without dependencies to the internet 
-the following online service was used: https://www.font-converter.net/en/css-embedded-font-base64
+The HTML page embedding all required styles, fonts and images for making the html page to run standalone without 
```

Comment:
Use upper case HTML consistently (in the beginning of the line, it's "HTML", later it's "html")

Reply:
done